policyengine_us/variables/gov/states/ct/tax/income/ct_income_tax_before_personal_credit.py

```diff
@@ -16,23 +16,19 @@ def formula(tax_unit, period, parameters):
         p = parameters(period).gov.states.ct.tax.income.rates
         itax_before_personal_credits = select(
             [
-                filing_status == status.SINGLE,
                 filing_status == status.JOINT,
                 filing_status == status.SEPARATE,
                 filing_status == status.SURVIVING_SPOUSE,
                 filing_status == status.HEAD_OF_HOUSEHOLD,
             ],
             [
-                p.single.calc(taxable_income),
                 p.joint.calc(taxable_income),
                 p.separate.calc(taxable_income),
                 p.surviving_spouse.calc(taxable_income),
                 p.head_of_household.calc(taxable_income),
             ],
+            default=p.single.calc(taxable_income),
         )
         add_back = tax_unit("ct_income_tax_phase_out_add_back", period)
         tax_recapture = tax_unit("ct_income_tax_recapture", period)
-        total = itax_before_personal_credits + add_back + tax_recapture
-        personal_credit_rate = tax_unit("ct_personal_credit_rate", period)
-        personal_credit_amount = personal_credit_rate * total
-        return max_(0, total - personal_credit_amount)
+        return max_(0, itax_before_personal_credits + add_back + tax_recapture)
```

Comment:
dont think we need the max statement here as taxable income as capped at 0

Reply:
ok this was based on previous logic, but refactored it to use an add statement in that case